src/utilities.cpp

```diff
@@ -0,0 +1,25 @@
+#include "utilities.hpp"
+
```

Comment:
These functions should be in their own namespace

Reply:
We can do that. Any particular reason though?

Reply:
I saw it often in other peoples code. I think it does not have an immediate benefit for us
- does not pollute the global namespace (we won't have that many functions)
- makes it clear that it's a user defined function: `webutils::split` (we won't have that many functions)
Just to play around with namespaces I guess :D

Reply:
Alright, sure, let's do it!

Reply:
ok